tests/sentry/api/path_params/resources/test_organization_slug.py

```diff
@@ -0,0 +1,378 @@
+from typing import Any
+from unittest.mock import MagicMock, patch
+
+from pytest import fixture
+from rest_framework.request import Request
+
+from sentry.testutils.cases import TestCase
+from sentry.testutils.helpers.features import with_feature
+
+from .test_id_or_slug_path_params_mixin import APIIdOrSlugTestMixin
+
+
+class OrganizationSlugTests(TestCase, APIIdOrSlugTestMixin):
+    @fixture(autouse=True)
+    def _mock_subdomain_is_region(self):
+        with patch("sentry.api.bases.organization.subdomain_is_region"):
+            yield
+
+    @patch("sentry.types.region.subdomain_is_region")
+    def organization_test(self, endpoint_class, slug_params, *args):
+        slug_kwargs = {param: self.slug_mappings[param].slug for param in slug_params}
+        id_kwargs = {param: self.slug_mappings[param].id for param in slug_params}
+
+        request = Request(request=self.make_request())
+
+        _, converted_slugs = endpoint_class().convert_args(request=request, **slug_kwargs)
+        _, converted_ids = endpoint_class().convert_args(request=request, **id_kwargs)
+
+        self.assert_conversion(endpoint_class, converted_slugs, converted_ids)
+
+    def region_organization_integration_test(self, endpoint_class, slug_params, *args):
+        slug_kwargs = {param: self.slug_mappings[param].slug for param in slug_params}
+        id_kwargs = {param: self.slug_mappings[param].id for param in slug_params}
+
+        non_slug_mappings: dict[str, Any] = {
+            "integration_id": 1,
+        }
+
+        request = Request(request=self.make_request())
+
+        _, converted_slugs = endpoint_class().convert_args(
+            request=request, **slug_kwargs, **non_slug_mappings
+        )
+        _, converted_ids = endpoint_class().convert_args(
+            request=request, **id_kwargs, **non_slug_mappings
+        )
+
+        self.assert_conversion(
+            endpoint_class,
+            converted_slugs,
+            converted_ids,
+            reverse_non_slug_mappings=non_slug_mappings,
```

Comment:
I'm a bit confused by this, why is a regular mapping being passed to a reverse one?

Reply:
It is because the convert_args doesn't change anything about the param. (Keeps the name as integration_id & its value as 1), so I decided not to create a second dict with the same value.

Reply:
Could you add a comment mentioning this?